juntagrico/views_subscription.py

```diff
@@ -224,6 +224,10 @@ def dispatch(self, request, cs_session, *args, **kwargs):
             logout(request)
             cs_session.clear()  # empty session object
 
+        # save sign up comment to session for admin notification => comment is not stored in member model
+        if request.method == 'POST' and hasattr(cs_session, 'comment'):
```

Comment:
Instead of this do 

    self.cs_session.main_member.comment = form.cleaned_data['comment']

in the `form_valid` method below, one line before the return

Reply:
fixed it

Reply:
fix is in new PR https://github.com/juntagrico/juntagrico/pull/450